### PR TITLE
fix(Badge): show icons in badge stories

### LINKED ISSUE #789: [Badge]: stories does not show icons
###  **Steps to reproduce**

1. Go to components -> Badge
2. Select Small Size
3. Go to stories tab
4. From controls, try selecting an icon

### Expected Behaviour
The selected icon should show up on the left side of the text inside the badge.

### Actual behaviour
Icon does not show up inside the badge.

Screenshot 2022-10-27 at 10 21 01 PM

**Note**: Same behaviour for `Medium Size`, `With Icon` and `Bold Font` stories.

### PR DESCRIPTION
**Pull request by shivam1646**
resolves #789

### CODE REVIEW

**Review by saurabhdaware — APPROVED** [review]
The icons seem to work for me on current storybook as well https://github.com/razorpay/blade/issues/789#issuecomment-1294452036.

Is it happening for any particular browser or device?

Either way this seems like a nice refactor to me 😄 (we can wait for one more review and merge if rest of the team agrees)

**Review by divyanshu013 — APPROVED** [review]
Looks good to me.

**Review by kamleshchandnani — APPROVED** [review]
🚀

**Pull request merged**
